lastmile-ai/aiconfig
# [5/n] Create vscode extension scaffolding

[5/n] Create vscode extension scaffolding


Just boilerplate code.

Test Plan:
* `yarn install && yarn compile`
* Run in Extension Development Host
* Open travel.aiconfig.json
* Make sure "Hello World from aiconfig-editor" is displayed

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1062).
* #1075
* #1073
* #1070
* #1069
* #1068
* #1066
* __->__ #1062
* #1060
* #1059
* #1058
* #1057
